### ISSUE TITLE
Syntax error in ipython-kernel-0.6.1.1's Haddock comments

### ISSUE THREAD

**Issue by peti** [gibiansky/IHaskell#550]
Citing from http://hydra.cryp.to/build/1055411/log/raw:

~~~~~~~
src/IHaskell/IPython/ZeroMQ.hs:146:23:
    parse error on input ‘-- ^ HMAC encryption key’
~~~~~~

**Comment by gibiansky** [comment]
Whoa, this makes no sense. That's not a syntax error!

Is this `haddock` output? What program is yielding the syntax error?

Look at [this line](https://github.com/gibiansky/IHaskell/blob/master/ipython-kernel/src/IHaskell/IPython/ZeroMQ.hs#L146) -- I think it's weird, but it's not wrong (and the package currently compiles fine on my box with `stack`).

I've uploaded a new version with a cleaned up syntax there, but this might be a bug in whatever program is doing the compiling there..?

**Comment by peti** [comment]
The error message comes from `Haddock version 2.16.1` as shipped with GHC 7.10.2 on Linux/x86_64.
